src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$main-size: 100px;
+$main-gap: 1px;
+$day-bg-color: #eee;
+$day-hover-bg-color: #ffbfcb;
+$day-names-in-week: (
+  'mon': 1,
+  'tue': 2,
+  'wed': 3,
+  'thu': 4,
+  'fri': 5,
+  'sat': 6,
+  'sun': 7,
+);
```

Comment:
The names of the days in the `$day-names-in-week` map should be full and descriptive to improve readability. Abbreviations can cause confusion and are not recommended according to the code style rules.